### PR TITLE
Modifications to basic templates.

### PR DESCRIPTION
**Pull request by ajparsons**
Adjusted title links, fixed social links, fixed line size of cc license.

### CODE REVIEW

File: repository/templates/base.html [hunk 1]
```diff
@@ -41,12 +41,12 @@
...
       <div class="container">
-          <a href="/" class="site-title"><h1>mySociety Research</h1></a>
+          <a href="{% url 'home' %}" class="site-title"><h1>mySociety Research</h1></a>
           <nav role="navigation" class="site-nav">
               <ul>
-                  <li><a href="{% url 'home' %}">Home</a></li>
-                  <li><a href="{% url 'page' 'strategy' %}">Research Strategy</a></li>
-                  <li><a href="{% url 'items' %}">Our Research</a></li>
+                  <li><a href="{% url 'items' %}">Publications</a></li>
+                  <li><a href="https://www.mysociety.org/category/research/">Blog</a></li>
```
Comment (jacksonj04) [inline repository/templates/base.html]: Might be worth checking with designers on if they have any views/thoughts on pushing people off-site in top-level navigation.

Comment (ajparsons) [inline repository/templates/base.html]: Are you thinking of the tictec or blog thing here?

Comment (jacksonj04) [inline repository/templates/base.html]: Both – I'm worried that it's an unexpected behaviour for a top-level link to take someone off-site.

Comment (ajparsons) [inline repository/templates/base.html]: Checked with martin, was cool with it.

Comment (jacksonj04) [inline repository/templates/base.html]: I'm happy to ship it then! :shipit:

**Pull request closed without merging**